tests/test_annotated.py

```diff
@@ -26,6 +26,11 @@
             5,
             id='annotated-field-value-default',
         ),
+        pytest.param(
+            lambda: Annotated[int, Field(..., description='Test')],
+            5,
```

Comment:
I makes no sense to me to have both `...` (i.e. make it required) in `Field` and `5` as default value 🤔 
Can you explain the usage?

Reply:
This was just me trying to make sure Annotated + Field with ellipsis worked on the test where you explicitly cover those. My usecase is the one from the other added test: being able to annotate Optional ("nullable") fields in a NamedDict so they get i.e. better descriptions, while keeping them required on the json schema. 

On looking at it again you're right it doesn't make sense to have ellipsis and a default, and accounting for not having a value set probably overcomplicates this test, so I'll just remove this.